src/eterna/folding/__tests__/Folder.test.ts

```diff
@@ -42,6 +42,32 @@ for (let folderType of [Vienna, Vienna2, NuPACK, LinearFoldV]) {
             .resolves.toBeUndefined();
     });
 
+    test('Nupack_Subopt_structures', () => {    
+   	 expect.assertions(1);
+    	 return expect(CreateFolder(NuPACK)
+           .then((folder) => {
+            if (folder === null) {
+                expect(true).toBeTruthy();
+                return;
+            }
+
+            const structures = folder.getSuboptEnsemble(
+                Sequence.fromSequenceString("auauauagaaaauauaua"),
+                37, 1, false);
+	    
+	        //let num = structures.length;
+	        console.log(structures); 
+            //console.log(structures[0]); 			 
+                
+            //expect(structures).toBeDefined();
+            //expect(structures[0])
+                //.toEqual(".((((((.....))))))");
```

Comment:
You should actually do some assertions here so that we can make sure future updates don't break anything. Also, remove any extra logging and dead code